src/components/RequestList/RequestItem/index.tsx

```diff
@@ -308,29 +299,20 @@ const RequestItem: React.FC<RequestItemProps> = ({
               )}
             </div>
             <div className="card-field">
-              <span className="card-field-name">
-                {intl.formatMessage(messages.requested)}
-              </span>
-              <span className="text-gray-300">
-                {intl.formatDate(requestData.createdAt, {
-                  year: 'numeric',
-                  month: 'long',
-                  day: 'numeric',
-                })}
-              </span>
-            </div>
-            <div className="card-field">
-              <span className="card-field-name">
-                {intl.formatMessage(messages.modified)}
-              </span>
-              <span className="truncate">
-                {requestData.modifiedBy ? (
-                  <span className="flex text-sm text-gray-300">
+              {hasPermission(
+                [Permission.MANAGE_REQUESTS, Permission.REQUEST_VIEW],
+                { type: 'or' }
+              ) ? (
+                <>
+                  <span className="card-field-name">
+                    {intl.formatMessage(messages.requested)}
+                  </span>
+                  <span className="flex text-sm text-gray-300 truncate">
                     {intl.formatMessage(messages.modifieduserdate, {
                       date: (
                         <FormattedRelativeTime
                           value={Math.floor(
-                            (new Date(requestData.updatedAt).getTime() -
+                            (new Date(requestData.createdAt).getTime() -
```

Comment:
Why was this changed to the creation date? This is when it was last modified?

Reply:
Oh I see we are still hiding the user who approved/declined it. I thought we decided to not do this?

Reply:
So what are we showing here? If the user has manage requests/view requests they will see the wrong date? And see who requested it?

What if people have multiple admins and want to see who is modifying requests? This will hide it.